<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab1</class>
 <widget class="QWidget" name="Tab1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>862</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tab1</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>711</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buyButton">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>420</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>购买</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="searhEdit">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>310</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>360</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>420</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CLIDBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="CIDBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>商品类别：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>商品新旧：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="messageButton">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>480</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>留言</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="market.qrc"/>
 </resources>
 <connections/>
</ui>
